src/Lean/Server/Watchdog.lean

```diff
@@ -514,6 +536,10 @@ section MessageHandling
           code := ErrorCode.internalError
           message := s!"Failed to process request {id}: {e}"
         }
+    if method == "textDocument/definition" || method == "textDocument/declaration" then
```

Comment:
A comment as to why we try to handle this in the watchdog and why we still need the worker sometimes would be helpful here